modules/tinymce/src/themes/silver/main/ts/ui/general/Button.ts

```diff
@@ -96,12 +96,8 @@ const calculateClassesFromButtonType = (buttonType: 'primary' | 'secondary' | 't
 export const renderButtonSpec = (spec: ButtonSpec, action: Optional<(comp: AlloyComponent) => void>, providersBackstage: UiFactoryBackstageProviders, extraBehaviours = [], extraClasses = []) => {
   const translatedText = providersBackstage.translate(spec.text);
 
-  const icon: Optional<AlloySpec> = spec.icon ? spec.icon.map((iconName) => renderIconFromPack(iconName, providersBackstage.icons)) : Optional.none();
```

Comment:
The `spec.icon` falsy check here was not needed as `spec.icon` is not optional and wasn't a boolean property.